### PR TITLE
PR679 + Audit Issue 8

### PR DESCRIPTION
**Pull request by john-0x**

### CODE REVIEW

File: chapters/exchanges.md [hunk 1]
```diff
@@ -505,10 +505,12 @@ onlyManager notShutDown`
 
 Please see parameter descriptions above. In the context of the Kyber exchange contract's nomenclature (swapToken() function), the following parameters take the specified mappings:
 
-  `orderAddresses[2]` - The Src token address.
-  `orderAddresses[3]` - The Dest token address.
-  `orderValues[0]` - The Src token quantity.
-  `orderValues[0]` - The Dest token quantity.
+  `orderAddresses[2]` - The Src (maker asset) token address.
```
Comment (jennazenk) [inline chapters/exchanges.md:508]: kyber adapter terminology has been changed to maker/taker

**Review by jennazenk — APPROVED** [review]

File: chapters/exchanges.md [hunk 2]
```diff
@@ -505,10 +505,12 @@ onlyManager notShutDown`
...
+  `orderAddresses[3]` - The Dest (taker asset) token address.
+  `orderValues[0]` - The Src (maker asset) token quantity.
+  `orderValues[0]` - The Dest (taker asset) token quantity.
```
Comment (GauthamN) [inline chapters/exchanges.md:511]: should this be orderValues[1]?

**Review by GauthamN — APPROVED** [review]

File: chapters/exchanges.md (hunk 2 above)
Comment (jennazenk) [reply chapters/exchanges.md:511]: @john-0x ?

Comment (jennazenk) [reply chapters/exchanges.md:511]: ah sorry saw the fix in other PR! all good!

**Pull request merged**